.travis.yml

```diff
@@ -1,11 +1,25 @@
 language: node_js
 node_js:
    - "node"
-after_script: "./node_modules/istanbul/lib/cli.js cover ./node_modules/mocha/bin/_mocha --report lcovonly -- --compilers js:babel-core/register -R spec && cat ./coverage/lcov.info | ./node_modules/coveralls/bin/coveralls.js && rm -rf ./coverage"
+
+addons:
+  chrome: stable
+
+sudo: required
+
+before_install:
+    - export PATH=$HOME/.local/bin:$PATH
+    - pip install --upgrade pip
+    - pip install --user crossbar
+    - sh -c 'crossbar start &'
+
+after_script: "./node_modules/istanbul/lib/cli.js cover ./node_modules/mocha/bin/_mocha --report lcovonly -- -R spec && cat ./coverage/lcov.info | ./node_modules/coveralls/bin/coveralls.js && rm -rf ./coverage"
```

Comment:
I feel, that ur repo is out of sync with current dev-branch. 'cause i've fixed coverall report in last commits...